### PR TITLE
Improve error message when a Svelte config is not found

### PR DESCRIPTION
**Pull request by ykomatsu**
`svelte-kit` error message can be a bit confusing when both `svelte.config.js` and `svelte.config.cjs` are absent.
We need `svelte.config.js` , not `svelte.config.cjs` :

```
$ npm run dev
Cannot find module '/Users/ykoma/Repositories/kit-playground/svelte.config.cjs' imported from /Users/ykoma/Repositories/kit-playground/node_modules/@sveltejs/kit/dist/cli.js
...
```

After this PR:

```
$ npm run dev
Cannot find module '/Users/ykoma/Repositories/kit-playground/svelte.config.js' imported from /Users/ykoma/Repositories/kit-playground/node_modules/@sveltejs/kit/dist/cli.js
...
```

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Review by dummdidumm — COMMENTED** [review]
Please provide a changeset, other than that this looks good 👍

File: .changeset/neat-points-exercise.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+Improve an error message for MODULE_NOT_FOUND
```
Comment (bluwy) [inline .changeset/neat-points-exercise.md:5]:
```suggestion
Improve an error message for MODULE_NOT_FOUND
```

Improve error message when a Svelte config is not found

**Review by bluwy — APPROVED** [review]

**Review by Rich-Harris — CHANGES_REQUESTED** [review]
Thank you! This still isn't a great error message though — it should be a much more explicit 'Missing svelte.config.js', or perhaps

> You need to create a svelte.config.js file — see https://kit.svelte.dev/docs#configuration

In fact, there's already some logic for that, it's just a) in a [weird place](https://github.com/sveltejs/kit/blob/b4aef9635c579087466d85819cea3294af948390/packages/kit/src/cli.js#L32-L39) that means it never actually runs, and b) overly complicated. We should just check to see if `svelte.config.js` or `svelte.config.cjs` exists, and if not throw a helpful error inside `load_config`

**Review by Rich-Harris — APPROVED** [review]

**Comment by Rich-Harris** [comment]
Thanks. Realised we can actually simplify this stuff quite a bit, because the checks for `svelte.config.cjs` and `vite.config.js` are from many months ago, and really aren't needed any more

**Pull request merged**